data/campaigns/Sceptre_of_Fire/scenarios/4t_The_Jeweler.cfg

```diff
@@ -300,7 +300,7 @@
         [/message]
         [message]
             speaker=Theganli
-            message= _ "Well... Maybe you can ask the Shorbear clan? They have good tools... Chisels and grit of some rare mineral, I don’t know what it is or how they got it, but their work is well known among us gem crafters."
+            message= _ "Well... Maybe you can ask the Shorbear clan? They have good tools... chisels... and grit of some rare mineral. I don’t know what they are or how they got them, but their work is well known among us gem crafters."
```

Comment:
> That sounds good to me. But we should probably run it by someone who understands the actual intent here, if possible.

> @nemaara please could you comment on this.

Other people who might understand the actual intent: @Wedge009 @Vultraz @doofus-01

Reply:
I think I made that change, intent was that all of it was made of some rare mineral, all the equipment is special and mysterious.  I was just trying to give some reason why the dwarves needed to borrow those particular things.  Maybe the string needs fixing, but not this way.  The dwarves aren't headed out to get special grit.

Reply:
Is it cool if I change it to this:
> They have good tools... chisels made of some rare mineral

https://dictionary.cambridge.org/grammar/british-grammar/made-from-made-of-made-out-of-made-with

Reply:
Well, it doesn't seem like an improvement to me, but I don't think it breaks anything.  No objections.

Reply:
I'm replaying this campaign and after reading this section, I wonder if the original author meant 'grit of some rare mineral' as a reference to diamonds, since diamond-tipped tools are often used to cut very hard materials in the real world. The context here is that Theganli the jeweller has just attempted to cut the ruby according to Thursagan's specifications but explains in the previous line that 'it can’t be cut, or scratched, or damaged at all! At least not by my tools...' Thus he makes the suggestion to visit the Shorbear clan because they good tools... -> the line in question here.

The PR as it current stands, with max changing it to 'chisels made of some rare mineral' the suggests that entire chisel is made of diamond. Well, I suppose we can assume that whatever the chisels are made of, they're a lot better than what Theganli has on hand - and at any rate, they're a plot device required for Rugnur and friends to pay them a visit in the next scenario. So I suppose the change is okay but it does change the meaning of the original intent, if indeed 'grit' was meant as an oblique reference to diamonds.

Feel free to mark the conversation as resolved again, if you don't think it's worth revisiting this aspect.

Reply:
It's fine in the revised version, I think.

Reply:
Marking this as resolved. There seems to be consensus.